Dockerfile

```diff
@@ -2,10 +2,10 @@ FROM openjdk:8u121
 
 RUN mkdir /static/public -p
 
-COPY ./src/app/static/public /static/public
+COPY ./app/static/public /static/public
```

Comment:
So this has changed because it's now happening within the build docker image?

Reply:
Exactly, but note that it's still used from `scripts/build-app.sh` which itself is called by `src/app/static/scripts/run-dependencies-for-integration-tests.sh` and should still work.